Exercise_D and E/assets/style/index.css

```diff
@@ -1,4 +1,89 @@
-/* 
-This css sheet is linked to practice.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+* {
+    margin: 0;
+    flex-direction: column;
+    max-width: 100%;
+    max-height: 100%;
+    position: relative;
+  }
+  
```

Comment:
Try to stick to one colour convention while you are developing.
When you work for a company you will be given a colour palette and the format they want you to code in.